packages/upgrade/src/commands/upgrade.js

```diff
@@ -90,6 +90,16 @@ export async function upgrade(options, availableUpgrades = []) {
       logger.log(packageJson.diff());
     }
   }
+
+  const migrations = upgrade.migrations;
+  if (migrations && migrations.length > 0) {
+    for (const migration of migrations) {
+      logger.verbose('running migration: %s', migration.name);
+
+      // Run each migrate function sequentially/non-async
+      migration.migrate(workspace.directory);
```

Comment:
```suggestion
      await migration.migrate(workspace.directory);
```

Does it make sense to run these in series as `async` chunks? In the case that something in the `migrate` function is async (like reading/writing files)

Reply:
@joshblack Yes, I'll also update the migrate command to do this as well